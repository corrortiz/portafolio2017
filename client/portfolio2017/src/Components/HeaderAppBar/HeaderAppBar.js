// @flow
import React from 'react';
import AppBar from 'material-ui/AppBar';
import Toolbar from 'material-ui/Toolbar';
import Typography from 'material-ui/Typography';
import { CircularProgress } from 'material-ui/Progress';

import TabMenu from '../TabMenu/TabMenu';
import TranslateMenu from '../TranslateMenu/TranslateMenu';

import GlobalsConnect from '../../HOC/GlobalsConnect/GlobalsConnect';

const HeaderAppBar = props => {
  return (
    <AppBar position="fixed" className="header">
      <Toolbar className="header__toolbar">
        <div className="header__title">
          <Typography
            type="title"
            color="inherit"
            className="header__title__message"
          >
            AO HyS
          </Typography>
          {props.globals.showLoading && (
            <CircularProgress
              size={24}
              className="header__title__progress"
              thickness={7}
            />
          )}
        </div>
        <TranslateMenu />
      </Toolbar>
      <TabMenu />
    </AppBar>
  );
};

export default GlobalsConnect(HeaderAppBar);
