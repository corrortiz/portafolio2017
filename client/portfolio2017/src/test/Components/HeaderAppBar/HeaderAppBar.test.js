import React from 'react';
import { shallow } from 'enzyme';
import HeaderAppBar from '../../../Components/HeaderAppBar/HeaderAppBar';

describe('HeadarAppBar', () => {
  it('should compare a snapshot of HeaderAppBar', () => {
    const wrapper = shallow(<HeaderAppBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
