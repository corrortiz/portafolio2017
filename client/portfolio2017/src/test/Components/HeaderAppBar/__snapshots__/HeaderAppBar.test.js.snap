// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadarAppBar should compare a snapshot of HeaderAppBar 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <withStyles(Connect(NameWrappedComponent)) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "flex": "Connect-NameWrappedComponent--flex-2",
        "menuButton": "Connect-NameWrappedComponent--menuButton-3",
        "root": "Connect-NameWrappedComponent--root-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "flex": "Connect-NameWrappedComponent--flex-2",
          "menuButton": "Connect-NameWrappedComponent--menuButton-3",
          "root": "Connect-NameWrappedComponent--root-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
